packages/strapi-plugin-content-manager/config/routes.json

```diff
@@ -188,6 +188,102 @@
           ["plugins::content-manager.hasPermissions", ["plugins::content-manager.explorer.publish"]]
         ]
       }
+    },
+    {
+      "method": "POST",
+      "path": "/collection-type/:model/:id/actions/lock",
```

Comment:
The routes will be `/collection-types` and `/single-types` to be more compliant with rest standards